app/generated_resources.grd

```diff
@@ -5983,6 +5989,18 @@ Keep your key file in a safe place. You will need it to create new versions of y
       <message name="IDS_PASSWORD_GENERATION_PROMPT_GOOGLE_PASSWORD_MANAGER" desc="Autofill dropdown text describing password generation.">
           You won't need to remember this password. It will be saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>user@gmail.com</ex></ph>.
       </message>
+      <message name="IDS_PASSWORD_GENERATION_HELP_TEXT_TRUSTED_ADVICE" desc="Autofill dropdown text describing password generation focusing on strong security messaging.">
+          Keep this account more secure with a strong password. It will be saved to <ph name="GOOGLE_PASSWORD_MANAGER">$1<ex>Brave Password Manager</ex></ph> for <ph name="EMAIL">$2<ex>user@gmail.com</ex></ph>.
```

Comment:
Same as below (RE: Brave Password Manager)

Reply:
The password manager redesign and password generation experiments are disabled upstream, but I entered https://github.com/brave/brave-browser/issues/30993 to track that and explicitly disable them with a feature override.

Reply:
I think we should definitely check out the password manager integration! 😄 cc: @rebron 

But we also just want to make sure branding shows Brave (which this does) and mentions sync for us (instead of signing into gmail account)

Reply:
We can mark these comments as resolved though, since feature is disabled 👍 Or we can consider it done w/ https://github.com/brave/brave-browser/issues/30993